data/nsmbu_intro.md

```diff
@@ -6,7 +6,7 @@ Thanks to [Kinnay](https://github.com/kinnay), [Luminyx](https://github.com/Lumi
 
 Command IDs (but not arguments, strangely) are empty in the static RPX data, and are filled in at runtime by the static init function at 0x021DAB60. Commands are read by the class with constructor at 0x021DC720 (unofficially, "CsEventMgr"). The `イベントアシスタント` ("Event Assistant") actor is also involved in some way, and might be responsible for executing the events.
 
-The scripts table is at 0x10044A60, and is an array of 119 of the following: `{uint32_t priority; void *script_ptr}`. Scripts are executed using a priority queue, so if multiple scripts are triggered at the same time, they'll execute in order of descending priority.
+The scripts table is at 0x10044A60, and is an array of 119 of the following: `{uint32_t priority; ScriptCommand *script_ptr}` with ScriptCommand being `{uint32_t type, uint32_t, arg}`. Scripts are executed using a priority queue, so if multiple scripts are triggered at the same time, they'll execute in order of descending priority.
```

Comment:
Thanks! I don't remember why I chose `void*`, so this is a good change. A few issues with the commit, though:

- The NSMB2 docs have almost the same wording as the NSMBU ones here, so it'd be best to make the same change to both for consistency.
- The struct format is already documented at the beginning of the next paragraph, so now it's redundant. Maybe you could just change the wording of the next paragraph instead ("The `ScriptCommand` struct has the same format as NSMBW [etc]")?
- It'd be good to briefly clarify that "`ScriptCommand`" is an unofficial name, e.g. "the `ScriptCommand` struct (unofficial name) has the same format as NSMBW [etc]"
- I think "`ScriptCommand`" should probably be in backticks to format it as code, though it's kind of debatable

Reply:
Hope I did this right...
[e37f359](https://github.com/RoadrunnerWMC/Cobra/pull/1/commits/e37f359df51715f045ad888c378659e04a5545e3)
[19db188](https://github.com/RoadrunnerWMC/Cobra/pull/1/commits/19db188751f90bb4af3af3a8e5e0e6320c5ddfa1)
[758a5e6](https://github.com/RoadrunnerWMC/Cobra/pull/1/commits/758a5e66ec1a70e01621d8818dc13e4aa5daab91)

Reply:
Thanks!